public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-admin.php

```diff
@@ -423,6 +423,15 @@ public static function meta_keys( $meta_group = '' ) {
 						'Global Sponsorship Grant'          => 'text',
 						'Running money through WPCS PBC'    => 'checkbox',
 					);
+
+					/*
+					 * The "Transparency Report Received" checkbox can only be checked or unchecked when the current user is admin or super admin.
+					 * See https://github.com/WordPress/wordcamp.org/issues/1280#issuecomment-2058571557.
+					 */
+					if ( current_user_can( 'manage_options' ) ) {
```

Comment:
Not sure if you've tested this scenario but I've come across several times in the past where a user who isn't show or displayed a field in this way can overwrite the field with a default value. Unsure if this is the case here, but wanted to mention it.

Reply:
Thanks for bringing that up! I hadn't thought of that case. 
Just checked off the field and then updated the post using another user unable to see the checkbox, and it didn't overwrite.